kernels/ZendEngine3/extended/fcall.c

```diff
@@ -278,7 +278,11 @@ static int zephir_is_callable_check_class(const char *name, int name_len, zend_f
 		} else {
 			fcc->called_scope = called_scope;
 			if (!fcc->object) {
-				fcc->object = Z_OBJ(EG(current_execute_data)->This);
+				#if PHP_VERSION_ID >= 70100
+					fcc->object = zend_get_this_object(EG(current_execute_data));
```

Comment:
Just in php 7.1 you can't use code I just changed it to this, check difference between those functions in php 7 and 7.1